geomstats/learning/pca.py

```diff
@@ -108,15 +107,15 @@ def __init__(self, metric, n_components=None, copy=True,
         self.iterated_power = iterated_power
         self.random_state = random_state
 
-    def fit(self, X,
```

Comment:
You removed the `base_point` argument (which defaults to the Fréchet mean), I assume this is because tPCA doesn't really make sense if you do not look at the Fréchet mean? But maybe its reasonable to keep it as an argument so that the user can supply the (already precalculated) Fréchet mean. Or is the computation time for the mean very small compared to the whole tPCA?

Reply:
You're right, the Frechet mean computation being an iterative algorithm that can take time, it is good to conserve this argument. Putting it back.